### ISSUE TITLE
Get a less rounded mz_delta value in ppm for annotations?

### ISSUE THREAD

**Issue by levitsky** [bittremieux/spectrum_utils#59]
Hi, thank you once again for this package. I am currently using it for some bulk annotations and, due to the rounding of ppm values to 1 decimal digit, I get very discrete values, which is not ideal for statistical analysis. Do you think it would be possible to enable more digits somehow?

I think this line is relevant:

https://github.com/bittremieux/spectrum_utils/blob/826f7701125086d7ed540337f1e351f232ac3bb3/spectrum_utils/spectrum.py#L736

Thank you!

**Comment by bittremieux** [comment]
Yes, that makes total sense. I initially used rounding to have some pretty plotting results,. But it's much better to do that in the plotting code then (and probably even have this behavior be user-configurable), and not use any rounding in the actual annotation.

**Comment by dikshant182004** [comment]
hi @bittremieux ;so we just need to remove the hard coded round funtion from the line or we need to add a user configurable option  (either through a function argument or a configuration setting) that lets users choose how many decimal places to display etc.

**Comment by bittremieux** [comment]
The appropriate fix is probably to not round in the line mentioned above, but only when [printing](https://github.com/bittremieux-lab/spectrum_utils/blob/826f7701125086d7ed540337f1e351f232ac3bb3/spectrum_utils/fragment_annotation.py#L197) or [plotting](https://github.com/bittremieux-lab/spectrum_utils/blob/826f7701125086d7ed540337f1e351f232ac3bb3/spectrum_utils/iplot.py#L67) the delta m/z values.

**Comment by dikshant182004** [comment]
okay got it .i will try to update it .
> should i make it user configurable(new funtion parameter) like when the user ask for ploting or printing then only it do rounding of values otherwise not .
> or just adjusted based on previous flow only with few changes

**Comment by gsreeja11** [comment]
If there isn't an PR already out for this, can I pick this up?
